### PR TITLE
JENKINS-35771# Fixed steps log API

### PR DESCRIPTION
**Pull request by vivek**
Related to issue # . 

Summary of this pull request: 
.
.
.

@reviewbybees

### CODE REVIEW

**Comment by blueoceandeploy** [comment]
Can one of the admins verify this patch?

File: blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineNodeTest.java [hunk 1]
```diff
@@ -622,6 +622,8 @@ public void getPipelineWihoutNodesAllStepsTest() throws Exception {
 
         List<Map> resp = get("/organizations/jenkins/pipelines/pipeline1/runs/1/steps/", List.class);
         Assert.assertEquals(4,resp.size());
+        String log = get("/organizations/jenkins/pipelines/pipeline1/runs/1/steps/"+resp.get(0).get("id")+"/log/", String.class);
+        System.out.println(log);
```
Comment (michaelneale) [inline blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineNodeTest.java]: Probably can remove this

**Comment by michaelneale** [comment]
:bee: :bee: :bee: 🍧

File: blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineNodeTest.java (hunk 1 above)
Comment (vivek) [inline blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineNodeTest.java]: Ah yes, was supposed to have assertion instead. fixed.

**Pull request merged**